draft-normington-jsonpath.md

```diff
@@ -297,15 +305,15 @@ Here is a complete overview and a side by side comparison of the JSONPath syntax
 
 | XPath | JSONPath           | Description                                                                                                                           |
 |-------+--------------------+---------------------------------------------------------------------------------------------------------------------------------------|
-| `/`   | `$`                | the root object/element                                                                                                               |
-| `.`   | `@`                | the current object/element                                                                                                            |
+| `/`   | `$`                | the root element/item                                                                                                                 |
```

Comment:
Why do we need "item"?  "Element" is strictly defined.  Maybe just use that.

Reply:
The slash is XML/JSON.  Element on the JSON side means something different than element on the XML side.

Reply:
Maybe that distinction could be made clearer (later).